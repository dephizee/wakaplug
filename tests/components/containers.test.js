import React from 'react';
import renderer from 'react-test-renderer';
import { AuthContainerStyled} from "./../../components/containers.style";


test('AuthContainer working', () => {
  const component = renderer.create(
    <AuthContainerStyled>Main container</AuthContainerStyled>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});