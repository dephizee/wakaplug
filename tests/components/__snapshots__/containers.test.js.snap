// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthContainer working 1`] = `
<div
  className="sc-bdvvtL cBlCT"
>
  Main container
</div>
`;
